### DISCUSSION TITLE
How do I validate someone has scrolled to the bottom of a document?

### DISCUSSION THREAD

**Discussion by willenglishiv** [jaredpalmer/formik#3187] [Q&A]
I'm sure there is some thing related to scroll position et. al but I'm trying to validate someone reading a TOS before they continue on.

**Comment by johnrom** [comment]
There are so many possible answers to this and it's a bit off topic for Formik.

I'm not endorsing a specific solution but here is an example of what you're looking for:

https://www.npmjs.com/package/react-bottom-scroll-listener

The callback in this case would simply set a state boolean like setIsDisabled(false) and that boolean would be passed to the disabled prop on your submit or continue button.
